west.yml

```diff
@@ -92,7 +92,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 4b493d4346e841e786a8daa2eb2ee03cbe5cfd37
+      revision: pull/105/head
```

Comment:
The hal PR has now been merged, so you can update this to the new SHA

Reply:
Updated